PendingReleaseNotes

```diff
@@ -43,3 +43,16 @@
   * system
   * truncated
   * user_stats_sync
+
+>=17.2.1
+
+* The "BlueStore zero block detection" feature (first introduced to Quincy in
+https://github.com/ceph/ceph/pull/43337) has been turned off by default with a
+new global config option called `bluestore_zero_block_detection`. This feature,
+intended for large-scale synthetic testing, does not interact well with some RBD
+and CephFS features. Any side affects experienced in previous Quincy versions would
+no longer occur, provided that the `bluestore_zero_block_detection` configuration
+remains set to false. Side effects that are now fixed include RBD thick-provisioned
+images ending up being thin-provisioned (apparent in `rbd du`) and incorrect output
+for object count vs. stored data (apparent in `ceph df`).
+Relevant tracker: https://tracker.ceph.com/issues/55521
```

Comment:
Can we make this a bit more concise? The linked tracker already has most of these details.